doc/dev/logging.md

```diff
@@ -0,0 +1,105 @@
+# Logging in operators
```

Comment:
@estroz Forgot one last nit. This should be in `doc/user/logging.md`
`doc/dev` is for developers of the SDK.